ajax/libs/React-Clipboard/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-clipboard.js",
+  "filename": "react-clipboard.min.js",
+  "version": "2.0.1",
+  "homepage": "https://github.com/nihey/react-clipboard.js",
```

Comment:
let's remove this line as we have repository info already, which points to the same place.